tests/contrib/molten/test_molten.py

```diff
@@ -57,22 +59,22 @@ def test_resources(self):
         expected_resources = [
             'molten.app.__call__',
             'molten.middleware.ResponseRendererMiddleware',
-            'molten.components.HeaderComponent',
-            'molten.components.CookiesComponent',
-            'molten.components.QueryParamComponent',
-            'molten.components.RequestBodyComponent',
-            'molten.components.RequestDataComponent',
-            'molten.components.SchemaComponent',
-            'molten.components.UploadedFileComponent',
-            'molten.components.HeaderComponent',
-            'molten.components.CookiesComponent',
-            'molten.components.QueryParamComponent',
-            'molten.components.RequestBodyComponent',
-            'molten.components.RequestDataComponent',
-            'molten.components.SchemaComponent',
-            'molten.components.UploadedFileComponent',
+            'molten.components.HeaderComponent.can_handle_parameter',
```

Comment:
A quick comment at L59 just on why some show up multiple times might be helpful, e.g. `Each component gets called for each view function argument` (or whatever the real reason is)